config/initializers/constants.rb

```diff
@@ -1,5 +1,5 @@
 TITLE_OPTIONS = %w(Mr Mrs Miss Ms)
-SUBSCRIPTION_REMINDER_TIME_DELTA = 4.hours
+SUBSCRIPTION_REMINDER_TIME_DELTA = 15.minutes
```

Comment:
wouldn't that variable affect prod as well ?

Reply:
Yup.. It will affect for prod as well but before merging to prod we will change the value. Or another thought is we can have 2 different constants for staging and prod. 


What are your thoughts @patrixr?